### ISSUE TITLE
Detect E2ETest/IntegrationTest Shutdown Crashes

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#6090]
> Should we add shutting down the intergratoin test app to see if there might be issues during shutdown?

_Originally posted by @dannyvv in https://github.com/microsoft/react-native-windows/pull/6087#discussion_r494416703_

We already collect crash dumps for these. We could maybe just add a step to shutdown, then fail CI if we see any crash dumps were recorded.

**Comment by chrisglein** [comment]
@kmelmon Is this something that's already happening for E2ETests? If the app crashes during the test, yes. But since the app is launched/terminated for each test case is this already covered?

**Comment by kmelmon** [comment]
@chrisglein yes each jasmine test spawns a separate instance of the test app, so what's remaining here is for Integration tests.  This is a nice-to-have.  Updating bug title and unassigning.
